### PR TITLE
Creating 2 commands to activate zoom tool and also to move between di…

### LINKED ISSUE #1095: Hotkeys not working: Z, PageUp, PageDown, =
## Issue 1 - Description:

Hotkeys not working on the Viewer page: `Z`, `PageUp`, `PageDown`.

### Steps to Reproduce

1. Open a study
2. Try to use the following hotkeys: `Z`, `PageUp`, `PageDown`
3. Checks that nothing happened.

### Actual Result

Hotkeys `Z`, `PageUp`, `PageDown` are not working, nothing happens at the viewer page.

### Expected Result

1. When user hits hotkey `Z`, the Zoom tool should be activated at the toolbar
2. User should be able to navigate on the thumbnails using hotkeys: `PageUp` and `PageDown`
2.1. When user hits hotkey `PageDown`, the next thumbnail series should be loaded in the viewport
2.2. When user hits hotkey `PageUp`, the previous thumbnail series should be loaded in the viewport

## Issue 2 - Description:

When there are multiple viewports, hotkey `=` does not reset the `inverted` status of the image in the active viewport.

In the gif below, user presses the `=` hotkey to reset all viewports before clicking on the `Reset` button

bug-hotkey-equal

### Steps to Reproduce

1. Open a study
2. Change layout to 3 viewports
3. Invert the image by pressing hotkey `I`
4. Press hotkey `=`

### Actual Result

Hotkey `=` does not reset the `inverted` status of the image. If user clicks on the `Reset` button, the `inverted` image will be reset with success.

### Expected Result

User should be able to reset the `inverted` image (in the active viewport) by pressing hotkey `=` or clicking on `Reset` button on toolbar.

### PR DESCRIPTION
**Pull request by galelis**
### Description
- Create missing command for `setZoomTool`
- Implement the command function for `updateViewportDisplaySet` that will be used to move (nexty/previous) into current loaded displaySets

Closes #1095

### CODE REVIEW

File: extensions/cornerstone/src/commandsModule.js [hunk 1]
```diff
@@ -334,24 +330,17 @@ const commandsModule = ({ servicesManager }) => {
...
-      options: { direction: -1 },
-    },
     // TOOLS
     setToolActive: {
       commandFn: actions.setToolActive,
       storeContexts: [],
       options: {},
     },
+    setZoomTool: {
+      commandFn: actions.setToolActive,
```
Comment (igoroctaviano) [inline extensions/cornerstone/src/commandsModule.js]: I think we can use the setToolActive in this case or we would go the path of having one command per tool

File: platform/viewer/src/appExtensions/GenericViewerCommands/commandsModule.js [hunk 2]
```diff
@@ -16,24 +12,63 @@ const actions = {
...
+  },
+  updateViewportDisplaySet: ({ viewports, direction }) => {
+    const viewportSpecificData = { ...viewports.viewportSpecificData };
+    const activeViewport = viewportSpecificData[viewports.activeViewportIndex];
+    const studyMetadata = utils.studyMetadataManager.get(
+      activeViewport.studyInstanceUid
+    );
+
+    if (!studyMetadata) {
+      return;
```
Comment (igoroctaviano) [inline platform/viewer/src/appExtensions/GenericViewerCommands/commandsModule.js]: @dannyrb we could create commands to wrap up state update logic which dispatches or a new action/reducer (redux) with a connected component? whats the model we should strive in this case?

File: platform/viewer/public/config/netlify.js [hunk 3]
```diff
@@ -53,12 +53,12 @@ window.config = {
     // firstImage
     // lastImage
     {
-      commandName: 'nextViewportDisplaySet',
```
Comment (igoroctaviano) [inline platform/viewer/public/config/netlify.js]: Next was previous, previous was next  🤓

Comment (galelis) [reply platform/viewer/public/config/netlify.js]: Yeah. Can you see any reason for that? hehe as it was like this everywhere it make me think about it. haha

File: extensions/cornerstone/src/commandsModule.js (hunk 1 above)
Comment (galelis) [reply extensions/cornerstone/src/commandsModule.js]: As `setZoomTool` is a hotkey command, hotkeys implementation does not support to runa command with options (yet!)

**Comment by igoroctaviano** [comment]
Looks like when I click length and activate the tool and then press Z, the icon changes and the tool is activated but it's not selected in the toolbar, is length instead. Maybe we should try to activate the tool in the toolbar as well? 

Besides that, it works. Left some small comments.

File: extensions/cornerstone/src/commandsModule.js (hunk 1 above)
Comment (dannyrb) [reply extensions/cornerstone/src/commandsModule.js]: I think we can keep this for now, but then lets create a follow-up issue where we allow hotkey definitions to include "options" that are passed to `runCommand`

Comment (galelis) [reply extensions/cornerstone/src/commandsModule.js]: I've create an issue: #1447

File: platform/viewer/src/appExtensions/GenericViewerCommands/commandsModule.js (hunk 2 above)
Comment (dannyrb) [reply platform/viewer/src/appExtensions/GenericViewerCommands/commandsModule.js]: I'm not sure I follow. Can you pseudo code an example?

**Pull request merged**